doc/variables.md

```diff
@@ -203,5 +204,17 @@ For more details see [scenario documentation](/doc/scenario.md).
 - `cartridge_fetch_backups` (`boolean`, default: `false`): flag indicates that backups should be
   fetched on the local machine;
 - `cartridge_fetch_backups_dir` (`string`, default: `backups/`): a directory on the local machine
-  where backups should be fetched if `cartridge_fetch_backups` is `true`. This path is relative to
+  where backups should be fetched if `cartridge_fetch_backups` is `true`; this path is relative to
   the playbook path.
+- `cartridge_restore_backup_path` (`string`): path to the instance backup archive on the remote
+  machine;
+- `cartridge_force_restore` (`boolean`, default: `false`): flag indicates that conflicting files
+  should be overwritten;
+- `cartridge_allow_alien_backup` (`boolean`, default: `false`): flag indicates that backup of
+  instance with another name can be used;
+- `cartridge_skip_cleanup_on_restore` (`boolean`, default: `false`): flag indicates that cleanup
+  before restoring should be skipped;
+- `cartridge_paths_to_keep_before_restore` (`list-of-strings`, default: `[]`): list of full paths or
+  relative paths to work/memtx/vinyl/wal directory that should be kept on instance
+  restore (`.tarantool.cookie` will be kept independently of this variable); it's possible to use
```

Comment:
"Should be kept" means that it wouldn't be removed before restore, or it wouldn't be rewritten by backup?